slapp/static/js/case_studies.js

```diff
@@ -268,3 +269,48 @@ function load_co2_chart(co2Data) {
         <span style="margin-left: 8px;">${text}</span>
     `;
 }
+
+// Update the sticky header with the region's title and image
+function updateStickyHeader(title) {
+  const titleEl = document.getElementById('sticky-region-title');
+  titleEl.textContent = title;
+}
+
+document.addEventListener('DOMContentLoaded', () => {
+  // 1) Determine navbar height to offset the sticky header
+  const navbar    = document.querySelector('.navbar__wrap');
+  const navHeight = navbar ? navbar.offsetHeight : 0;
+
+  // Apply that offset so sticky-header sits just below main nav
+  const stickyHeader = document.getElementById('sticky-region-header');
+  stickyHeader.style.top = `${navHeight}px`;
+
+  // 2) Bind click events to all select-buttons
+  document.querySelectorAll('.select-button').forEach(button => {
+    button.addEventListener('click', event => {
+      const wrapper = event.currentTarget.closest('.cs__region-container');
+      const title   = wrapper.dataset.regionTitle;
+      const bbox    = JSON.parse(wrapper.dataset.regionBbox);
```

Comment:
Unfortunately, I get an error in this line.
As there is no `regionBbox` in `wrapper.dataset` ?!